Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -46,6 +59,7 @@
             "label": "Version Behavior",
```

Comment:
we could also create groupings for version related inputs and for docker related inputs

Reply:
I assume your implying here that you could then configure both sets of inputs with one instance of the task.  But like I mentioned in my response to one of Nathan's comments, that would require that we introduce additional setting that allow the user to indicate whether they want to have either or both sets of settings applied, checkboxes presumably.  In that case, the user could potentially have both checkboxes unchecked which essentially leads to a no-op.  That just seems like a bad design.  To me, it just seems simpler to configure the task to do one action and if you need both actions, add another instance of the task.

---
In reply to: [136702938](https://github.com/Microsoft/vsts-tasks/pull/5076#discussion_r136702938) [](ancestors = 136702938)

Reply:
I was talking about different groups for docker related inputs and version related inputs. And then visibility of group can be controlled using updateType value